pyproject.toml

```diff
@@ -120,6 +119,9 @@ kubernetes = [
 pydantic = [
         "pydantic>=1.10.0,<2.0.0",
 ]
+aws = [
+    "apache-airflow-providers-amazon>=8.0.0",
+]
```

Comment:
Since, ATM, this dependency is specific to Athena, I suggest we add it to the previously existing:
```
dbt-athena = [
```